simpleclient_eclipselink/src/main/java/io/prometheus/client/eclipselink/EclipseLinkStatisticsCollector.java

```diff
@@ -0,0 +1,744 @@
+                Object value = metrics.get(SessionProfiler.RemoteChangeSet);
+                if (value != null) {
+                  return ((Integer) value).doubleValue();
+                }
+                return 0;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_client_sessions_created_total",
+            "Total number of client sessions",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.ClientSessionCreated);
+                if (value != null) {
+                  return ((Long) value).doubleValue();
+                }
+                return 0L;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_client_sessions_released_total",
+            "Total number of released client sessions",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.ClientSessionReleased);
+                if (value != null) {
+                  return ((Long) value).doubleValue();
+                }
+                return 0L;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_connects_total",
+            "Total number of connects",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.Connects);
+                if (value != null) {
+                  return ((Long) value).doubleValue();
+                }
+                return 0L;
+              }
+            }
+        )
+        ,
+        createCounter(
+            "eclipselink_disconnects_total",
+            "Total number of disconnects",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.Disconnects);
+                if (value != null) {
+                  return ((Integer) value).doubleValue();
+                }
+                return 0;
+              }
+            }
+
+        ),
+        createCounter(
+            "eclipselink_optimistic_lock_exceptions_total",
+            "Total number of optimistic lock exceptions",
+            new ValueProvider() {
+              @Override
+              public double getValue(Map<String, Object> metrics) {
+                Object value = metrics.get(SessionProfiler.OptimisticLockException);
+                if (value != null) {
+                  return ((Integer) value).doubleValue();
+                }
+                return 0;
+              }
+            }
+        ),
+        createCounter(
+            "eclipselink_rcm_received_total",
```

Comment:
What's rcm?

Reply:
RemoteCommandManager (https://www.eclipse.org/eclipselink/api/2.2/org/eclipse/persistence/sessions/coordination/RemoteCommandManager.html). I can also spell out the whole thing in the metric name.